index.html

```diff
@@ -98,8 +98,7 @@
 &nbsp; <A
 HREF="mpeuni/mmtheorems.html">Theorem list</A>
 &nbsp;
-<A HREF="http://us2.metamath.org:88/mpeuni/mmrecent.html">Recent proofs</A>
-<A HREF="mpeuni/mmrecent.html">(this mirror)</A>
+<A HREF="mpeuni/mmrecent.html">Recent proofs</A>
```

Comment:
```suggestion
<A HREF="http://us.metamath.org/mpeuni/mmrecent.html">Recent proofs</A>
<A HREF="mpeuni/mmrecent.html">(this mirror)</A>
```
This distinction is still relevant for mirror sites, like cn.metamath.org, as well as local copies of the site.